roles/mysql/vars/main.yml

```diff
@@ -33,4 +33,5 @@ mysql_configuration_options:
 
 mysql_configuration_destination: /etc/mysql/mariadb.conf.d
 
-mariadb_repo: "http://sfo1.mirrors.digitalocean.com/mariadb/repo/10.3/ubuntu"
+mariadb_repo: "https://deb.mariadb.org"
+mariadb_repo_version: 10.6
```

Comment:
Jammy runs mariadb 10.6. During the upgrade process, we want prod and staging to stay on 10.3.

We might want to set this in `roles/mysql/defaults/main.yml` so it will be overridden by any other setting.